Taskfile.yaml

```diff
@@ -20,13 +26,13 @@ tasks:
     dir: "{{.SRC_DIR}}"
     cmds:
       - test -z "$(gofumpt -d -e . | tee /dev/stderr)"
-      - go vet ./...
```

Comment:
`go vet` is included in `golangci-lint` https://golangci-lint.run/usage/linters/